apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.js

```diff
@@ -1,27 +1,50 @@
 // @ts-check
-const ERR = require('async-stacktrace');
 import * as express from 'express';
 import * as path from 'path';
 const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const asyncHandler = require('express-async-handler');
+import { z } from 'zod';
+import * as sqldb from '@prairielearn/postgres';
 
 import { config } from '../../lib/config';
-import * as sqldb from '@prairielearn/postgres';
 
 const router = express.Router();
 const sql = sqldb.loadSqlEquiv(__filename);
 
-router.get('/', function (req, res, next) {
-  debug('GET /');
-  var params = {
-    assessment_id: res.locals.assessment.id,
-    link_exam_id: config.syncExamIdAccessRules,
-  };
-  sqldb.query(sql.assessment_access_rules, params, function (err, result) {
-    if (ERR(err, next)) return;
-    res.locals.access_rules = result.rows;
+const assessmentAccessRulesSchema = z.object({
+  mode: z.string(),
+  uids: z.string(),
+  start_date: z.string(),
+  end_date: z.string(),
+  credit: z.string(),
+  time_limit: z.string(),
+  password: z.string(),
+  exam_uuid: z.string().nullable(),
+  ps_exam_id: z.string().nullable(),
+  pt_course_id: z.string().nullable(),
+  pt_course_name: z.string().nullable(),
+  pt_exam_id: z.string().nullable(),
+  pt_exam_name: z.string().nullable(),
+  active: z.string(),
+});
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    debug('GET /');
```

Comment:
Nit: you can remove the `debug` prints (this one and the one below) while you're here, I don't think they're particularly useful.

Reply:
Removed the `debug` prints in commit fce8031.